extensions/blocks/dialogue/edit.js

```diff
@@ -0,0 +1,260 @@
+	 * It handles local and global (conversation) level.
+	 *
+	 * @param {string} style - style to toggle.
+	 * @returns {void}
+	 */
+	function toggleParticipantStyle( style ) {
+		if ( isCustomParticipant || ! participantsFromContext ) {
+			return setAttributes( { [ style ]: ! attributes[ style ] } );
+		}
+
+		transcritionBridge.updateParticipants( {
+			participantSlug: currentParticipantSlug,
+			[ style ]: ! currentParticipant[ style ],
+		} );
+	}
+
+	/**
+	 * Helper to build the CSS classes for the participant label.
+	 * It handles local and global (conversation) level.
+	 *
+	 * @returns {string} Participant CSS class.
+	 */
+	function getParticipantLabelClass() {
+		if ( isCustomParticipant || ! participantsFromContext ) {
+			return classnames( `${ baseClassName }__participant`, {
+				[ 'has-bold-style' ]: attributes?.hasBoldStyle,
+				[ 'has-italic-style' ]: attributes?.hasItalicStyle,
+				[ 'has-uppercase-style' ]: attributes?.hasUppercaseStyle,
+			} );
+		}
+
+		return classnames( `${ baseClassName }__participant`, {
+			[ 'has-bold-style' ]: currentParticipant?.hasBoldStyle,
+			[ 'has-italic-style' ]: currentParticipant?.hasItalicStyle,
+			[ 'has-uppercase-style' ]: currentParticipant?.hasUppercaseStyle,
+		} );
+	}
+
+	return (
+		<div className={ className }>
+			<BlockControls>
+				{ currentParticipant && isFocusedOnParticipantLabel && (
+					<ToolbarGroup>
+						<ToolbarButton
+							icon="editor-bold"
+							isPressed={ hasStyle( 'hasBoldStyle' ) }
+							onClick={ () => toggleParticipantStyle( 'hasBoldStyle' ) }
+						/>
+
+						<ToolbarButton
+							icon="editor-italic"
+							isPressed={ hasStyle( 'hasItalicStyle' ) }
+							onClick={ () => toggleParticipantStyle( 'hasItalicStyle' ) }
+						/>
+
+						<ToolbarButton
+							icon={ formatUppercase }
+							isPressed={ hasStyle( 'hasUppercaseStyle' ) }
+							onClick={ () => toggleParticipantStyle( 'hasUppercaseStyle' ) }
+						/>
+					</ToolbarGroup>
+				) }
+			</BlockControls>
+
+			<InspectorControls>
+				<Panel>
+					<PanelBody title={ __( 'Participant', 'jetpack' ) }>
+						<ParticipantsControl
+							className={ baseClassName }
+							participants={ participants }
+							currentParticipantSlug={ participantSlug }
+							onSelect={ setAttributes }
+						/>
+					</PanelBody>
+
+					<PanelBody title={ __( 'Time stamp', 'jetpack' ) }>
+						<ToggleControl
+							label={ __( 'Show', 'jetpack' ) }
+							checked={ showTimeStamp }
+							onChange={
+								( show ) => transcritionBridge.setAttributes( { showTimeStamp: show } )
+							}
+						/>
+
+						{ showTimeStamp && (
+							<TimeStampControl
+								className={ baseClassName }
+								value={ timeStamp }
+								onChange={ ( newTimeStampValue ) => {
+									setAttributes( { timeStamp: newTimeStampValue } );
+								} }
+							/>
+						) }
+					</PanelBody>
+				</Panel>
+			</InspectorControls>
+
+			<div class={ `${ baseClassName }__meta` }>
```

Comment:
Change `class` to `className`

Reply:
fixing...

Reply:
fixed 7420e180ca02073f68ad3be38f6a59ed118584f6